src/olympia/abuse/models.py

```diff
@@ -233,6 +233,7 @@ class AbuseReport(ModelBase):
             'DSA: It violates the law or contains content that violates the law',
         ),
         ('POLICY_VIOLATION', 13, "DSA: It violates Mozilla's Add-on Policies"),
+        ('SOMETHING_ELSE', 14, 'DSA: Something else'),
```

Comment:
Should we use a high number like 127 again, given this is another "other" option?

Reply:
I thought about it but I don't see the point - grouping with other DSA values also has merit...